### PR TITLE
fix(helper): handle case where type is in an `anyOf` key

### PR DESCRIPTION
**Pull request by AntoineDao**
Small fix that enables the openapi helper to run succesfully for the Pollination-Server

### CODE REVIEW

File: pydantic_openapi_helper/helper.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ def set_format(p):
     """
     if '$ref' in p:
         return p
+    elif 'anyOf' in p:
+        return p
```
Comment (mostaphaRoudsari) [inline pydantic_openapi_helper/helper.py:28]: Hi @AntoineDao, an object with `anyOf` should not end up here. Am I going o hit this error if I run the code for the Pollination server?

Comment (AntoineDao) [reply pydantic_openapi_helper/helper.py:28]: Yes. Here is an example of an `anyOf` appearing in the pollination server OpenAPI definition but originally sourced from `queenbee`:

image

Comment (mostaphaRoudsari) [reply pydantic_openapi_helper/helper.py:28]: This was a bug in Pydantic (https://github.com/samuelcolvin/pydantic/issues/1209) which should be fixed now (https://github.com/samuelcolvin/pydantic/pull/1438). I wonder if you are using an older version of Pydantic to generate these docs?

**Comment by AntoineDao** [comment]
Confirmed that I wasn't using the right version of pydantic 😀

**Pull request closed without merging**